src/scope.ml

```diff
@@ -120,6 +120,68 @@ type mode =
       carries the environment for variables that will be bound in the
       representation of the RHS. *)
 
+(** [get_params_implicitness ss env t] returns a list representing the
+    formal parameters of a parser term [t] *)
```

Comment:
The comment is not very clear. The function returns a list of booleans. What's the meaning of this list?

Reply:
Clarified to :
[get_params_implicitness_of_type t] returns a list representing the
    implicitness of the parameters of a parser term [t] seen as a type